app/controllers/projects_controller.rb

```diff
@@ -37,7 +37,11 @@ def toggle_archive
   # PATCH /projects/1/toggle_locked.js
   def toggle_locked
     @project = Project.find(params[:id])
```

Comment:
One quick improvement is to make use of the `find_project` before action instead of duplicating code here.